desktop/renderer-app/src/utils/hooks/useURLAppLauncher.ts

```diff
@@ -0,0 +1,34 @@
+import { useContext, useEffect } from "react";
+import { matchPath, useHistory, useLocation } from "react-router-dom";
+import { GlobalStoreContext } from "../../components/StoreProvider";
+import { joinRoomHandler } from "../../pages/utils/joinRoomHandler";
+import { routeConfig } from "../../route-config";
+import { ipcReceive, ipcReceiveRemove } from "../ipc";
+import { usePushHistory, RouteNameType } from "../routes";
+/** using ipcReceive method listen to join room request event,
```

Comment:
import { usePushHistory, RouteNameType } from "../routes";

/** using ipcReceive method listen to join room request event,